network-api/networkapi/mozfest/templates/fragments/nav-links.html

```diff
@@ -1,8 +1,7 @@
 {% load primary_active_nav %}
 
-{{ pre }}<a class="{% primary_active_nav request '/spaces' %}" href="/spaces">Spaces</a>{{ post }}
-{{ pre }}<a class="{% primary_active_nav request '/team' %}" href="/team">Team</a>{{ post }}
-{{ pre }}<a class="{% primary_active_nav request '/tickets' %}" href="/tickets">Tickets</a>{{ post }}
-{{ pre }}<a class="{% primary_active_nav request '/sponsors' %}" href="/sponsors">Sponsors</a>{{ post }}
+{% for item in menu_items %}
+  {{ pre }}<a class="{% primary_active_nav request item.slug %}" href="{{ item.url }}">{{ item.title }}</a>{{ post }}
```

Comment:
General question: Any idea what the deal is with using `pre` and `post` here vs pulling those `div` elements into the markup here?

Actual request: the `active` class doesn't seem to get applied when using the [review app](https://foundation-mofostaging-pr-3294.herokuapp.com/en/mozilla-festival/spaces/). I haven't spun it up locally to check if it does when the festival is on its own domain, but ideally it should work both ways.

Reply:
Yeah:

https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/mozfest/templates/partials/primary_mozfest_nav.html#L15

vs.

https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/mozfest/templates/partials/primary_mozfest_nav.html#L37

One of them need `<div>` wrapping, The other does not.